chain/stmgr/forks.go

```diff
@@ -218,6 +223,11 @@ func runPreMigration(ctx context.Context, sm *StateManager, fn PreMigrationFunc,
 	height := ts.Height()
 	parent := ts.ParentState()
 
+	if disabled := os.Getenv(EnvDisablePreMigrations); strings.TrimSpace(disabled) == "1" {
+		log.Warn("SKIPPING pre-migration", "height", height)
+		return
+	}
+
```

Comment:
My original thought was to just set the `PreMigration` array for each migration to empty during the construction of the state manager when this environment var is set. This would lead to nothing getting scheduled.

If we had a way to print the current schedule / next scheduled task I think this would still be the correct way to go. However, I don't believe we have that functionality. So instead I'm opting to just exit pre-migrations early.

This will ensure that a log message is created around the time the pre-migration was supposed to run stating that it was skipped. I think this will make debugging things a bit easier

Reply:
I think this is a pretty good way to go about it, honestly.